Yosemite/Yosemite/Actions/CardPresentPaymentAction.swift

```diff
@@ -4,6 +4,31 @@
 import Combine
 
 public enum CardPresentPaymentAction: Action {
+    /// Switches the store to use WCPay as the backend. This is also the default.
+    ///
+    case useWCPay
+
+    /// Switches the store to use Stripe as the backend
+    ///
+    case useStripe
+
+    /// Retrieves and stores payment gateway account(s) for the provided `siteID`
+    /// We support payment gateway accounts for both the WooCommerce Payments extension AND
+    /// the Stripe extension. Let's attempt to load each and update view storage with the results.
+    /// Calls the passed completion with success after both loads have been attempted.
+    ///
+    case loadAccounts(siteID: Int64, onCompletion: (Result<Void, Error>) -> Void)
+
+    /// Get a Stripe Customer for an order.
+    ///
+    case fetchOrderCustomer(siteID: Int64, orderID: Int64, onCompletion: (Result<WCPayCustomer, Error>) -> Void)
```

Comment:
We might want to revisit the naming of WCPayCustomer in the future?

Reply:
Opened https://github.com/woocommerce/woocommerce-ios/issues/5903